commissioning.tex

```diff
@@ -33,14 +33,21 @@ \subsection{Template generation during commissioning}
 The commissioning period provides an excellent opportunity to investigate how many visits in a given band are sufficient to construct a usable template.
 Given the desire to maximize the science harvest prior to the \drone,  relaxing these criteria is an option to be explored.
 
+
 \subsection{Alert generation during commissioning}
 
 Due to the need to verify the instrument characteristics, template quality, and image differencing and Real/Bogus performance, real-time alerts will not be immediately available during the commissioning period.
-Where the accumulated ComCam data is sufficient for alert generation, we expect to provide alerts only after a delay expected to be weeks to months. 
-The goal for these {\it canned} alerts is to enable alert brokers and science users to understand their characteristics and to help to validate their quality rather than to enable rapid followup and \es per se.
-Templates generated during commissioning will be used for Alert Production, with the goal of delivering real-time alert distribution to community brokers by the time of the \svs at the end of LSSTCam commissioning.
+However, the incremental template generation and alert production processes will be tested and optimized during ComCam and LSST Cam on-sky commissioning, resulting in a set of prompt products and alerts.
+
```

Comment:
Good, thanks - I have been thinking of them as separate because the Alert DB is different from the PP DB (and I had thought that the Prompt Product Database did not contain Alerts, etc). I'll adopt your wording.

Reply:
This is all correct , but we still class them all in the category of "Prompt Data Products" (LPM-231)